lrlex/examples/calclex/src/main.rs

```diff
@@ -17,7 +17,7 @@ fn main() {
             Some(Ok(ref l)) => {
```

Comment:
Typo in commit message: `This commit thus change the API from:`

Reply:
Agreed.